schema.graphql

```diff
@@ -27705,6 +27652,11 @@ enum ProjectNextFieldType {
   """
   SINGLE_SELECT @deprecated(reason: "The `ProjectNext` API is deprecated in favour of the more capable `ProjectV2` API. Follow the ProjectV2 guide at https://github.blog/changelog/2022-06-23-the-new-github-issues-june-23rd-update/, to find a suitable replacement. Removal on 2022-10-01 UTC.")
 
+  """
```

Comment:
**⚠️ This is a breaking change**

The switch from "Tracks" to "Tasks" is a breaking change that is in the process of being reverted within GitHub.

**This change is not safe to merge. We should wait for it to be reverted before we merge the schema changes.**

Reply:
Schemas are up to date with this again: https://github.com/octokit/graphql-schema/commit/2fd00362d227765452022cf3c7339a681bf11458